src/common/color.helper.ts

```diff
@@ -33,7 +35,7 @@ export class ColorHelper {
     let colorScale;
     if (type === 'quantile') {
       colorScale = scaleQuantile()
-        .range(scheme.domain)
+        .range(scheme.domain as unknown as number[])
```

Comment:
`tslint` threw errors there because you were trying to put a `string[]` inside a `number[]`. Since I assume you guys know what you're doing, I added this recommended conversion. It will have absolutely no effect on the code, but it raises a red flag.